greynoise/plugin.spec.yaml

```
@@ -0,0 +1,509 @@
+---
+
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: greynoise
+title: GreyNoise
+vendor: greynoise
+support: community
+status: []
+description: Interact with GreyNoise API
```

Would be good to add a 1-2 sentence short description that highlights what the integration does. E..g The GreyNoise plugin allows users to ...

```suggestion
description: Interact with GreyNoise API
```